pfcli/service/client/deployment.py

```diff
@@ -28,9 +28,9 @@ def scale_deployment(self, deployment_id: str, replicas: int) -> None:
             err_prefix=f"Deployment ({deployment_id}) cannot be scaled.",
         )(pk=deployment_id, path=f"scale", data=str(replicas))
 
-    def list_deployments(self, project_id: str) -> Dict[str, Any]:
+    def list_deployments(self, project_id: str, archived: Optional[bool]) -> Dict[str, Any]:
```

Comment:
```suggestion
    def list_deployments(self, project_id: str, archived: Optional[bool] = None) -> Dict[str, Any]:
```
You want to send also `None` value to server?

Reply:
I found out sending `None` is treated as sending `False`.
So, I changed the type from Optional[bool] to bool.